roles/kubernetes/preinstall/tasks/set_facts.yml

```diff
@@ -25,7 +25,7 @@
 - set_fact: etcd_access_address="{{ access_ip | default(etcd_address) }}"
 - set_fact: etcd_peer_url="http://{{ etcd_access_address }}:2380"
 - set_fact: etcd_client_url="http://{{ etcd_access_address }}:2379"
-- set_fact: etcd_authority="{{ access_ip|default('127.0.0.1') }}:2379"
```

Comment:
This was written for openstack deployment. access_ip is the public floating IP for the vm, which is actually not a locally allocated IP address. It can't bind to it. Anyway, we should always use proxy on localhost.